README.md

```diff
@@ -27,6 +27,19 @@ Quick note, it is considered a good practice to use Google before contacting men
 
 For any troubles you're having with the organization, mentors or other students, feel free to contact the GCI admins (details below).
 
+## Don’t do these things
+- Don't use foul language to any student or mentor in any case. Always be polite to other community members and mentors.
+- You should never argue with mentors with regarding your work . They are highly experienced people if they ask you to do some change   you should follow their advice.
+- At times it might happen that there could be some delay with mentors reviewing your work. You should always consider the fact that      mentors are also humans they also have life and other work to do.You should *NOT* be constantly texting them to respond.
+- **WARNING**  Never try to copy someone else’s work .
+
+## Do these things
+- When Stuck in a task you should always try to find out the solution to the problem by googling it out before pinging mentors.
+- Try to have the discussions related to a project in the project's chatroom.
```

Comment:
We won't have dedicated chat rooms for projects, right @KeenWarrior

Reply:
I think we should make new chat rooms for the projects, it would make it much more organised. CC @KeenWarrior

Reply:
@aliok we may later decide to have multiple rooms. But I wouldn't suggest to do that in the initial run.

Reply:
Okay so, in that case, I will remove this line for the time being.